js/index.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+
+	$('#entry').submit(doTheThing);
+	// when the user submits the form with the ID entry, do the thing
+
+	function doTheThing(event) {
```

Comment:
While `doTheThing` is a fun name here, if this were a real site, it would not be an ideal function name. We want to make sure our function names are as descriptive as possible. Here a better name might be something like `changeBackgroundCity` or something like that.